docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: config.clear_cart_link.url %}
```

Comment:
The return_to is optional, we can add it to the section below where we described the optional param.